src/main/scala/nl/knaw/dans/easy/stage/lib/CSV.scala

```diff
@@ -36,7 +36,7 @@ class CSV private (csv: Seq[Seq[String]], requiredHeaders: Seq[String]) {
   def getRows: Seq[Seq[String]] =
     csv.tail.map(value => actualHeaders.zip(value)
       // ignore columns with headers that are not options
-      .filter(t => requiredHeaders.contains(t._1))
+      .filter(t => requiredHeaders.contains(t._1) && t._2.nonEmpty)
```

Comment:
what are `t._1` and `t._2` here? Not very descriptive. Maybe you can do something like this: `.filter { case (x, y) => ...)` where `x` and `y` are placeholders for more useful names.